spec/support/api/helpers.rb

```diff
@@ -22,7 +22,7 @@ def init_api_spec_env
         def api_basic_authorize(*identifiers, user: @user.userid, password: @user.password)
           if identifiers.present?
             identifiers.flatten.collect do |identifier|
-              @role.miq_product_features << MiqProductFeature.find_or_create_by(:identifier => identifier)
+              @role.miq_product_features << MiqProductFeature.find_or_create_by(:identifier => identifier) if identifier
```

Comment:
`find_or_create_by`?!  Why is this code creating product features?  @abellotti @gtanzillo ?